Sources/SwiftProtobuf/Errors.swift

```diff
@@ -52,6 +56,8 @@ public enum DecodingError: Error {
     case malformedProtobuf
     /// Field names were not compiled into the binary
     case missingFieldNames
+    /// The field identifier (name or number) was not recognized
+    case unknownField
     /// TODO: More here?
```

Comment:
Slight aside - does it make sense to have different error enums for each of the different decoders so it's a little more clear what error can happen for a given operation?